whois-api/src/test/java/net/ripe/db/whois/api/rest/WhoisSearchServiceTestIntegration.java

```diff
@@ -527,6 +527,49 @@ public void search_inverse() {
         ));
     }
 
+    @Test
+    public void search_inverse_sponsoring_org() {
```

Comment:
Belongs in other PR

Reply:
check the description
Depends on https://github.com/RIPE-NCC/whois/pull/1431

This will disappear from here once the other PR is merged